com.unity.render-pipelines.high-definition/Runtime/PostProcessing/PostProcessSystem.cs

```diff
@@ -1817,7 +1817,14 @@ void DoMotionBlur(CommandBuffer cmd, HDCamera camera, RTHandle source, RTHandle
             using (new ProfilingScope(cmd, ProfilingSampler.Get(HDProfileId.MotionBlurKernel)))
             {
                 cs = m_Resources.shaders.motionBlurCS;
+
+                if (m_EnableAlpha)
+                {
+                    cs.EnableKeyword("ENABLE_ALPHA");
```

Comment:
you need to call explicitely a 

if (m_EnableAlpha)
                {
                    cs.EnableKeyword("ENABLE_ALPHA");
}
else
{
          cs.DisableKeyword("ENABLE_ALPHA");
}

otherwise you can get in trouble.
This is not done in other part of postprocess 
due to this line:
 m_FinalPassMaterial.shaderKeywords = null;

But better, in core utils , please add:

        public static void SetKeyword(ComputeShader cs, string keyword, bool state)
        {
            if (state)
                cs.EnableKeyword(keyword);
            else
                cs.DisableKeyword(keyword);
        }

Then call
CoreUtils.SetKeyword(cmd, "ENABLE_ALPHA", m_EnableAlpha);